TheJumpCloud/support
# Removal of remote assist install/uninstall commands

## Issues
* [TX-409](https://jumpcloud.atlassian.net/browse/TX-409) - Remove RAA cmds from support repo

## What does this solve?
Removes the Remote Assist install and uninstall commands from the commands gallery 

## Is there anything particularly tricky?
No

## How should this be tested?
Verify that references to the remote assist install and uninstall scripts are removed from the commands gallery

## Screenshots


[TX-409]: https://jumpcloud.atlassian.net/browse/TX-409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
